nikola/plugins/task/scale_images.py

```diff
@@ -45,6 +45,7 @@ def set_site(self, site):
 
     def process_tree(self, src, dst):
         """Process all images in a src tree and put the (possibly) rescaled images in the dst folder."""
+        thumb_fmt = self.kw['image_thumbnail_format']
         ignore = set(['.svn'])
```

Comment:
Yes, I considered that, but I don't like retrieving the same item from a dictionary over and over again during each iteration, that's why I opted for retrieving it once before entering the loop.